packages/tasks/src/pipelines.ts

```diff
@@ -676,6 +676,11 @@ export const PIPELINE_DATA = {
 		color: "red",
 		hideInDatasets: true,
 	},
+	"document-retrieval": {
```

Comment:
Sounds a bit NLP-y to me. Could we use something like "visual-document-retrieval", for symmetry with "visual-question-answering"?